src/main/resources/lib/credentials/store.jelly

```diff
@@ -0,0 +1,57 @@
+<!--
+The MIT License
+
+Copyright 2025 Jan Faracik
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
+    <st:documentation>
+        A reusable Store card.
+    </st:documentation>
+
+    <div class="credentials-card">
+        <div class="credentials-card__inner credentials-card__inner--split">
+            <div class="credentials-card__title">
+                <l:icon src="${store.iconClassName}" class="jenkins-avatar" tooltip="${store.provider.displayName}"/>
+                <a href="${storeAction==null?store.relativeLinkToAction:store.relativeLinkToContext+'credentials/store/'+storeAction.urlName}">
+                    ${store.context == app ? store.displayName : store.contextDisplayName}
+                </a>
+            </div>
+            <div class="credentials-card__tags">
+                ${%Domains:}
+                <j:forEach var="domain" items="${store.domains}">
+                    <a href="${storeAction==null?store.getRelativeLinkTo(domain):store.relativeLinkToContext+'credentials/store/'+storeAction.urlName+'/'+domain.url}"
+                       class="jenkins-badge">
+                        <j:choose>
+                            <j:when test="${domain.global}">
+                                ${%global}
```

Comment:
nearly illegible for me in Firefox
<img width="947" height="250" alt="Screenshot from 2025-12-30 12-40-23" src="https://github.com/user-attachments/assets/ff389d9f-bb04-4104-ba14-5daf08f40794" />